### PR TITLE
updates

### PR DESCRIPTION
**Pull request by linobanana**
added authorization page, booking and success confirmation component, responsive header

### CODE REVIEW

File: GUI/src/app/app.component.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+body {
```
Comment (EvgeniZhibrik) [inline GUI/src/app/app.component.css]: why body styles are here? and what these margins for?

File: GUI/src/app/app.component.html [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <div *ngIf="isAppReady">
   <app-header></app-header>
-  <router-outlet></router-outlet>
```
Comment (EvgeniZhibrik) [inline GUI/src/app/app.component.html]: please, get router-outlet back

File: GUI/src/app/pages/authorization-page/authorization-page.component.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+  <div class="form-container">
+    <form class="authorization-form" [formGroup]="authorizationForm" (ngSubmit)="onSubmit()">
+      <h2 class="form-label">User Authorization</h2>
```
Comment (EvgeniZhibrik) [inline GUI/src/app/pages/authorization-page/authorization-page.component.html]: Please, move all string constants to constant or enum in shared folder

File: GUI/src/app/pages/authorization-page/authorization-page.component.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import {FormControl, FormGroup, Validators} from '@angular/forms';
+
+@Component({
+  selector: 'app-authorization-page',
+  templateUrl: './authorization-page.component.html',
+  styleUrls: ['./authorization-page.component.css']
+})
+export class AuthorizationPageComponent {
+  authorizationForm = new FormGroup({
```
Comment (EvgeniZhibrik) [inline GUI/src/app/pages/authorization-page/authorization-page.component.ts]: please, move initialization to constructor + use FormBuilder

File: GUI/src/app/app.component.css (hunk 1 above)
Comment (linobanana) [reply GUI/src/app/app.component.css]: Yes, it's my fault, I forgot delete it

File: GUI/src/app/app.component.html (hunk 2 above)
Comment (linobanana) [reply GUI/src/app/app.component.html]: done

File: GUI/src/app/pages/authorization-page/authorization-page.component.html (hunk 3 above)
Comment (linobanana) [reply GUI/src/app/pages/authorization-page/authorization-page.component.html]: done

File: GUI/src/app/pages/authorization-page/authorization-page.component.ts (hunk 4 above)
Comment (linobanana) [reply GUI/src/app/pages/authorization-page/authorization-page.component.ts]: done

**Pull request merged**